python/cudf_polars/tests/containers/test_dataframe.py

```diff
@@ -141,3 +142,8 @@ def test_sorted_flags_preserved(with_nulls, nulls_last):
     assert b.null_order == b_null_order
     assert c.is_sorted == plc.types.Sorted.NO
     assert df.flags == gf.to_polars().flags
+
+
+def test_empty_name_roundtrips():
+    df = pl.LazyFrame({"": [1, 2, 3], "column_0": [4, 5, 6]})
```

Comment:
Do we also want to test the case where the other column's name wouldn't conflict by default (IIUC in this case the empty would be naturally renamed to column_0 as well, triggering two separate issues)? This seems like a unit test covering two cases at once.

Reply:
Done.